crates/starknet-server/tests/test_messaging.rs

```diff
@@ -481,6 +481,19 @@ mod test_messaging {
             devnet.post_json("/postman/flush".into(), "".into()).await.expect("flush failed");
         assert_eq!(resp.status(), StatusCode::OK, "Checking status of {resp:?}");
 
+        // Assert traces of L1Handler transaction with custom rpc call,
```

Comment:
We already have a pretty long test case here, adding this is only making it longer. Could this addition be a separate test, like `assert_l1_handler_tx_can_be_dumped_and_loaded`? If not, at least L487-L492 could be extracted into a separate function for a slight improvement.

Reply:
4a15d01c319ef3ca8c2321a44b705767cf584cda